### DISCUSSION TITLE
Best Upgrade Path

### DISCUSSION THREAD

**Discussion by lookme2** [inventree/InvenTree#7770] [General]
What is the best upgrade path from 0.8.4 to the current latest version?  I've attempted to go straight from 8.4 to 15.7 which did not seem to work.  I've looked through the documentation and I don't really see anything about upgrading from a really old version.

**Comment by SchrodingersGat** [comment]
As a starting point please provide the full error log when you try to upgrade.

Also what platform are you on, docker etc

**Reply by lookme2** [reply]
Sorry about the delay.  We got this figured out.  We ended up just doing a fresh install and copying the DB over.  Did a invoke migrate and it all seems to work well.  Thanks!
